apps/wallet-ui/src/lib/assets/erc20.ts

```diff
@@ -0,0 +1,72 @@
+import type { Erc20Token } from "@/types/Erc20Token";
```

Comment:
probably add `import "server-only";` since I assume this will only be called as server action / on backend?